### PR TITLE
Small Grammar Update

### PR DESCRIPTION
**Pull request by sunhwapark**
Line 000499

He is > He's + grammar change

Line 000501,000502,000506
Changed to match the quest item name; Blue Dyes > Blue Paint 
Although, 501 needs to be taken a look at.

Line 000508

I am > I'm

sentence alteration
I am > I'm  + some word changes

Line 000503
I am > I'm

Line 00505
grammar change
'the' > 'that'

Line 000506
'You would be able to obtain' > ' You can obtain'

Line 000512
Typo Fix 
'fro' > 'from' 

Line 000514
sentence fixing

Line 000518

The Current in-game quest item is named 'Dark Blue Herb', changed to match it.
blue black herb > Dark Blue Herb 

Line 000521
grammar
'to your eyes' > 'in your eyes' 
'would' > 'will'

### CODE REVIEW

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -8666,27 +8666,27 @@ QUEST_LV_0100_20151001_008665	What happened?{nl}I think I've heard a some noise
 QUEST_LV_0100_20151001_008666	Illina... was attacked?{nl}How can Ignas do something without going crazy!{nl}
 QUEST_LV_0100_20151001_008667	I will follow the words of Lydia Schaffen on behalf of Illina.{nl}I will arrive at Orsha no matter what it takes.{nl}
 QUEST_LV_0100_20151001_008668	To you... I want to give you some rewards since you've protected me until the end.{nl}I am sure Illina and Lydia Schaffen also want me to do that.
-QUEST_LV_0100_20151001_008669	Please feed that black herb to Agatas.{nl}As it gets detoxified, the weak body of a human will suffer severe pains, but that will be better than dying.{nl}
+QUEST_LV_0100_20151001_008669	Please Feed the Dark Blue Herbs to Agatas.{nl}As it gets detoxified, the weak body of a human will suffer severe pains, but that will be better than dying.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:8669]: Feed -> feed

Comment (sunhwapark) [inline EnglishTranslation-master/QUEST_LV_0100.tsv:8669]: Thanks, changes have been applied

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**